src/aof.c

```diff
@@ -2196,6 +2196,103 @@ static int rewriteFunctions(rio *aof) {
     return 0;
 }
 
+int rewriteSelectDbRio(rio *aof, int db_num) {
+    char selectcmd[] = "*2\r\n$6\r\nSELECT\r\n";
+    if (rioWrite(aof, selectcmd, sizeof(selectcmd) - 1) == 0) return 0;
+    if (rioWriteBulkLongLong(aof, db_num) == 0) return 0;
+    return 1;
+}
+
+int rewriteObjectRio(rio *aof, robj *o, int db_num) {
```

Comment:
nit - `bool` is recommended 
```suggestion
bool rewriteObjectRio(rio *aof, robj *o, int db_num) {
```

Reply:
Done